system-addon/lib/SectionsManager.jsm

```diff
@@ -40,13 +40,10 @@ const BUILT_IN_SECTIONS = {
       },
       button: {id: options.disclaimer_buttontext || "section_disclaimer_topstories_buttontext"}
     },
+    privacyNoticeURL: "https://www.mozilla.org/en-US/privacy/firefox/#suggest-relevant-content",
```

Comment:
@Mardak @rlr Yes, let's remove info_link from topstories.options and add privacy_link instead to make this configurable. We can also delete provider_header (from options) and the corresponding string pocket_feedback_header now that we're no longer using it.

Reply:
If you prefer not having to configure this link we can add: 
`options.privacy_link || "https://www.mozilla.org/en-US/privacy/firefox/#suggest-relevant-content"`

This way we can override in case we ever need to, but it will work by default.

Reply:
makes sense. will fix. thanks!